handlers/secrets.go

```diff
@@ -51,7 +51,8 @@ func MakeSecretsHandler(c client.SecretAPIClient) http.HandlerFunc {
 			responseStatus, responseBody, responseErr = createNewSecret(c, body)
 			break
 		case http.MethodPut:
-			responseStatus, responseBody, responseErr = updateSecret(c, body)
+			responseStatus = http.StatusMethodNotAllowed
+			responseErr = fmt.Errorf("docker swarm secrets are immutable")
```

Comment:
`faas-swarm is unable to update secrets, delete and re-create or use a new name`

Reply:
Please could you add a unit test for this scenario if we don't have one yet?

Reply:
We already had unit test for this scenario, I have updated it.